gif-screencast.el

```diff
@@ -230,7 +295,7 @@ A screenshot is taken before every command runs."
 
 (defun gif-screencast-print-status (process event)
   "Output PROCESS EVENT to minibuffer."
-  (princ (format "Process '%s' %s" process event)))
+  (princ (format "Process '%s' %s" process (s-chomp event))))
```

Comment:
I'd rather not add an extra dependency just for one function call.  gif-screencast does almost no string processing at all, I think it's more reasonable to chomp ourselves.

Reply:
Yes, right. I agree with your comment.
I'll provide a short code for this and test it in Ubuntu and macOS :)

Reply:
Why testing on 2 OSes?  This has nothing to do with the operating-system level, has it?

Reply:
I believe the returned value of `event` depends on each process which is generated by `start-process` and the process might be different between Operating Systems. But anyway, I checked it in both systems :p